lastpasslib/secrets.py

```diff
@@ -186,6 +187,27 @@ def add_attachment(self, attachment):
         """
         self._attachments.append(attachment)
 
+    def delete(self):
+        """Deletes the secret from Lastpass."""
+        url = f'{self._lastpass.host}/show.php'
+        data = {
+            'aid': self.id,
+            'delete': '1',
+            'encuser': self._lastpass.encrypted_username,
+            'requesthash': self._lastpass.encrypted_username,
+            'sentms': f"{time.time_ns() // 1_000_000}",
+            'token': self._lastpass.token
+        }
+        response = self._lastpass.session.post(url, data=data)
+        if not response.ok:
+            response.raise_for_status()
```

Comment:
The conditional is not required, this will only raise if there is a not ok response.